### DISCUSSION TITLE
can one ceph cluster shared multi k8s cluster，and how to do it？

### DISCUSSION THREAD

**Discussion by 13567436138** [rook/rook#13591] [Q&A] [answered]
can one ceph cluster shared multi k8s cluster，and how to do it？
for example ceph cluster is installed by cephadm，and shared by two other k8s cluster by rook

**Comment by subhamkrai** [comment, marked as answer]
@13567436138 please go through this, this will https://rook.github.io/docs/rook/latest/CRDs/Cluster/external-cluster/?h=exter it will help
